tests/test_dask_image/test_ndmeasure/test_core.py

```diff
@@ -110,7 +112,12 @@ def test_center_of_mass(datatype):
     ]
 )
 def test_measure_props(funcname, shape, chunks, has_lbls, ind):
-    sp_func = getattr(spnd, funcname)
+    if funcname == 'sum_labels' and scipy.__version__ < LooseVersion('1.5.0'):
```

Comment:
```suggestion
    if funcname == 'sum_labels' and scipy.__version__ > LooseVersion('1.5.0'):
```
It is my fault :cry: